tests/providers/google/cloud/transfers/test_gcs_to_gcs.py

```diff
@@ -86,6 +86,19 @@ def test_execute_no_suffix(self, mock_hook):
             TEST_BUCKET, prefix="test_object", delimiter=""
         )
 
+    @mock.patch('airflow.providers.google.cloud.transfers.gcs_to_gcs.GCSHook')
+    def test_execute_wildcard_with_replace_flag_false(self, mock_hook):
+        operator = GCSToGCSOperator(
+            task_id=TASK_ID, source_bucket=TEST_BUCKET,
+            source_object=SOURCE_OBJECT_WILDCARD_SUFFIX,
+            destination_bucket=DESTINATION_BUCKET,
+            replace=False)
+
+        operator.execute(None)
+        mock_hook.return_value.list.assert_called_once_with(
+            TEST_BUCKET, prefix="test_object", delimiter="", replace=False
+        )
```

Comment:
This method is called two times during execute: L282 and L288 so we should do something like
```python
calls = [
    mock.Call(TEST_BUCKET, prefix="test_object", delimiter=""),
    mock.Call(DESTINATION_BUCKET, prefix="test_object", delimiter=""),
]
mock_hook.return_value.assert_has_calls(calls)
```

Reply:
Thanks!